CONTRIBUTING.md

```diff
@@ -0,0 +1,56 @@
+# Contributing to the Eclipse Che4z LSP for HLASM project
+
+Thanks for your interest in this project.
+
+## Eclipse Che4z
+
+This project is a part of [Eclipse Che4z](https://projects.eclipse.org/projects/ecd.che.che4z) which provides components/extensions for Eclipse Che to facilitate mainframe application development.
+
+## Eclipse Contributor Agreement
+
+Before your contribution can be accepted by the project team contributors must electronically sign the Eclipse Contributor Agreement (ECA).
```

Comment:
I don't understand this sentence.

Reply:
Is this better?
Before your contribution can be accepted by the project team, you must electronically sign the Eclipse Contributor Agreement (ECA).

Reply:
Perfect. Or simply:
```suggestion
Before your contribution can be accepted by the project team, contributors must electronically sign the Eclipse Contributor Agreement (ECA).
```